rudder-web/src/main/scala/com/normation/rudder/web/components/popup/CreateRulePopup.scala

```diff
@@ -183,9 +183,11 @@ class CreateOrCloneRulePopup(
   private[this] def error(msg:String) = <span class="col-lg-12 errors-container">{msg}</span>
 
   private[this] def closePopup() : JsCmd = {
-      JsRaw("""
-        $('#createRulePopup').bsModal('hide');
-        $('#settingsTab').bsTab('show');
+      JsRaw(s"""
+        $$('#createRulePopup').bsModal('hide');
+        if(!${clonedRule.isDefined}){
```

Comment:
I think we should just remove the bs tab. 

The problem her eis that the tab changed even we did not want